.github/workflows/linters.yml

```diff
@@ -77,6 +77,14 @@ jobs:
         run: |
           make lint-py-build || true
           NODE=$(which node) make lint-py
```

Comment:
```suggestion
          NODE=$(command -v node) make lint-py
```

Reply:
Do you want to open a PR which fixes that? I feel that doesn't quite belong to this one, and there are a few other occurence of `which` that we can replace in our `Makefile`.

Reply:
Cool, I'll open a different PR to replace the `which`s. :slightly_smiling_face: